module/zfs/metaslab.c

```diff
@@ -3311,7 +3311,7 @@ metaslab_alloc_dva(spa_t *spa, metaslab_class_t *mc, uint64_t psize,
 	 * For testing, make some blocks above a certain size be gang blocks.
 	 * This will also test spilling from special to normal.
 	 */
-	if (psize >= metaslab_force_ganging && (ddi_get_lbolt() & 3) == 0) {
+	if (psize >= metaslab_force_ganging && (spa_get_random(100) < 3)) {
```

Comment:
Can you add to the comment above, noting that this also causes device removal to create more split blocks, and if this is turned up too high, then reconstruction of ztest-induced damage can take too long?

Reply:
Done.